src/components/BoardReaction/BoardReaction.tsx

```diff
@@ -0,0 +1,31 @@
+import {BoardReactionType, BoardReactionImageMap} from "types/reaction";
+import {memo} from "react";
+import {getRandomNumberInRange} from "utils/random";
+import {useAppSelector} from "store";
+import classNames from "classnames";
+import "./BoardReaction.scss";
+
+type BoardReactionProps = {
+  reaction: BoardReactionType;
+};
+
+// component that displays a board reaction (emoji + name)
+// memo prevents re-rendering even when parent state changes
+export const BoardReaction = memo((props: BoardReactionProps) => {
+  const emoji = BoardReactionImageMap.get(props.reaction.reactionType);
+  const me = useAppSelector((state) => state.participants!.self);
+  const others = useAppSelector((state) => state.participants!.others);
+  const all = [me, ...others];
+  const reactionUser = all.find((p) => p.user.id === props.reaction.user)!;
+  const {name} = reactionUser.user;
+  const reactedSelf = reactionUser.user.id === me.user.id;
+
+  const leftOffset = getRandomNumberInRange(5, 85);
```

Comment:
On mobile, the board reactions are sometimes cut off be the right side of the screen. It's not bad, but might be worth to take a look at
<img width="202" alt="Bildschirmfoto 2023-09-28 um 09 40 44" src="https://github.com/inovex/scrumlr.io/assets/35272402/a0c68e43-a148-45c0-aa42-d6314a0790a2">

Reply:
you could do the offset purely in scss (which imo would be better anyway) and then change the value using a media query

Reply:
> you could do the offset purely in scss (which imo would be better anyway) and then change the value using a media query

what should the behaviour be in this case? The overflow is visible right now, leading to a scrollbar which is ugly. Changing the overflow to hidden just cuts it off, which is a little better but for long names still annoying.  
Since the offset of the board reactions is choosen at random, maybe we could take the name length into the equasion  to thwart the overflow?  
 
> you could do the offset purely in scss (which imo would be better anyway) and then change the value using a media query

there's the random function in scss, but isn't the number fixed by the time of compilation? so the reactions would still always  appear at the same place

Reply:
see [this entry](https://blog.codepen.io/2013/08/26/random-function-in-sass/#random-at-runtime)

Reply:
Oh yeah, you're right about the random numbers in scss, my bad.
About the overflow: Is it possible to just decrease the leftOffset for mobile users so that reactions don't appear so far to the right of the screen? I think taking the name into account would be a bit overkill here

Reply:
it should now take the size of the container into account, so it's never cut off 👍